src/main/java/net/glowstone/dispenser/ArmorDispenseBehavior.java

```diff
@@ -0,0 +1,56 @@
+package net.glowstone.dispenser;
+
+import java.util.ArrayList;
+import net.glowstone.block.GlowBlock;
+import net.glowstone.block.blocktype.BlockDispenser;
+import org.bukkit.Material;
+import org.bukkit.block.BlockFace;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.Location;
+import org.bukkit.World;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.entity.Entity;
+import java.util.List;
+import org.bukkit.entity.Player;
+import net.glowstone.inventory.GlowInventory;
+
+public class ArmorDispenseBehavior extends DefaultDispenseBehavior {
+    DefaultDispenseBehavior defaultBehavior = new DefaultDispenseBehavior();
+    
+    @Override
+    protected ItemStack dispenseStack(GlowBlock block, ItemStack stack) {
+        Material armor = stack.getType();
+        BlockFace facing = BlockDispenser.getFacing(block);
+        GlowBlock target = block.getRelative(facing);
+        Location location1 = target.getLocation();
+        Location location2 = location1.clone();
+        location2.setY(location2.getY() - 1);
+        World world = location1.getWorld();
+        
+        //Find all nearby entities and see if they are players or armor stands
+        List<LivingEntity> entities = new ArrayList<>();
+        for (Entity cEntity : world.getNearbyEntities(location1, 3, 3, 3)) {
```

Comment:
I think I used to have a variable called entity, so I put a `c` infront of it for `currentEntity`. I'll change it to entity if you like.